### PR TITLE
empty table and null aggregations fix

### PR DESCRIPTION
**Pull request by egor-ryashin**

### CODE REVIEW

File: runtime/server/queries_columns.go [hunk 1]
```diff
@@ -479,11 +488,17 @@ func (s *Server) GetTimeRangeSummary(ctx context.Context, request *runtimev1.Get
...
+		if v := rowMap["min"]; v != nil {
+			summary.Min = timestamppb.New(v.(time.Time))
+			summary.Max = timestamppb.New(rowMap["max"].(time.Time))
+			summary.Interval, err = handleInterval(rowMap["interval"])
+			if err != nil {
+				return nil, err
+			}
+		} else {
+			summary.Min = timestamppb.Now()
+			summary.Max = timestamppb.Now()
```
Comment (begelundmuller) [inline runtime/server/queries_columns.go:500]: Shouldn't they be `nil` in this case?

File: runtime/server/queries_columns.go [hunk 2]
```diff
@@ -479,11 +488,17 @@ func (s *Server) GetTimeRangeSummary(ctx context.Context, request *runtimev1.Get
...
+			summary.Interval = &runtimev1.TimeRangeSummary_Interval{}
```
Comment (begelundmuller) [inline runtime/server/queries_columns.go:501]: Shouldn't this also be `nil`?

**Review by begelundmuller — CHANGES_REQUESTED** [review]
Thanks for these fixes! Added some questions below. CI is also failing – can you take a look?

File: runtime/server/queries_columns.go (hunk 1 above)
Comment (egor-ryashin) [reply runtime/server/queries_columns.go:500]: I wonder what should be the behaviour of functions that depend on it, let's consider we need to return an aggregation on an empty table and the time range object contains null values, should we return timeseries values array as nil or arrays should be empty?

**Review by begelundmuller — APPROVED** [review]
lgtm

**Pull request merged**